tokens/fatcat.json

```diff
@@ -0,0 +1,11 @@
CAxMDEuODM1OTM4IEMgOTYuMTYwMTU2IDEwMS41ODIwMzEgOTYuMTYwMTU2IDEwMS41ODIwMzEgOTUuODgyODEyIDEwMS4zNTkzNzUgQyA5NS42NzU3ODEgMTAxLjEwOTM3NSA5NS42NzU3ODEgMTAxLjEwOTM3NSA5NS42ODM1OTQgMTAwLjc5Mjk2OSBDIDk1LjcxMDkzOCAxMDAuNjc1NzgxIDk1LjczODI4MSAxMDAuNTU4NTk0IDk1Ljc2NTYyNSAxMDAuNDM3NSBDIDk1Ljg4MjgxMiA5OS41NzAzMTIgOTUuNzA3MDMxIDk5LjA5Mzc1IDk1LjE5NTMxMiA5OC4zOTA2MjUgQyA5NC44NjMyODEgOTguMDExNzE5IDk0LjUyMzQzOCA5Ny42MzY3MTkgOTQuMTc1NzgxIDk3LjI2OTUzMSBDIDkzLjUyNzM0NCA5Ni41ODU5MzggOTIuOTIxODc1IDk1Ljg3MTA5NCA5Mi4zMjAzMTIgOTUuMTQ0NTMxIEMgOTIuMjM0Mzc1IDk1LjAzOTA2MiA5Mi4xNDg0MzggOTQuOTMzNTk0IDkyLjA1ODU5NCA5NC44MjQyMTkgQyA5MC43MTA5MzggOTMuMTYwMTU2IDkwLjcxMDkzOCA5My4xNjAxNTYgOTAuNzEwOTM4IDkyLjE1MjM0NCBaIE0gOTAuNzE4NzUgOTIuMTYwMTU2ICIvPgo8cGF0aCBzdHlsZT0iIHN0cm9rZTpub25lO2ZpbGwtcnVsZTpub256ZXJvO2ZpbGw6cmdiKDc3LjY0NzA1OSUsNTUuNjg2Mjc1JSwyMS41Njg2MjclKTtmaWxsLW9wYWNpdHk6MTsiIGQ9Ik0gMTM4LjM5ODQzOCA2MS42MDE1NjIgQyAxMzkuODAwNzgxIDYxLjcyMjY1NiAxNDEuMTgzNTk0IDYxLjkyNTc4MSAxNDIuNTc0MjE5IDYyLjE2MDE1NiBDIDE0Mi45NDUzMTIgNjIuMjIyNjU2IDE0My4zMTY0MDYgNjIuMjgxMjUgMTQzLjY4NzUgNjIuMzQzNzUgQyAxNDUuNjg3NSA2Mi42NzE4NzUgMTQ3LjY1NjI1IDYzLjA4MjAzMSAxNDkuNTU4NTk0IDYzLjc1NzgxMiBDIDE0OS45NzI2NTYgNjMuODkwNjI1IDE1MC4zODY3MTkgNjMuOTQ5MjE5IDE1MC44MTI1IDY0LjAxMTcxOSBMIDE1MS4xMzI4MTIgNjQuMTcxODc1IEMgMTUxLjEwMTU2MiA2NC4yNjU2MjUgMTUxLjA3MDMxMiA2NC4zNjMyODEgMTUxLjAzOTA2MiA2NC40NTcwMzEgQyAxNTAuOTI5Njg4IDY1LjA0Njg3NSAxNTEuMjYxNzE5IDY1LjUwNzgxMiAxNTEuNTIzNDM4IDY2LjAyMzQzOCBDIDE1MS42NDA2MjUgNjYuMjY5NTMxIDE1MS43NjE3MTkgNjYuNTE1NjI1IDE1MS44ODI4MTIgNjYuNzYxNzE5IEMgMTUxLjk0NTMxMiA2Ni44OTA2MjUgMTUyLjAwNzgxMiA2Ny4wMTU2MjUgMTUyLjA3MDMxMiA2Ny4xNDg0MzggQyAxNTMuNTI3MzQ0IDcwLjIwMzEyNSAxNTQuNTgyMDMxIDczLjQxNzk2OSAxNTUuNDQ1MzEyIDc2LjY5OTIxOSBDIDE1NS42NDA2MjUgNzcuNDQ1MzEyIDE1NS44NTE1NjIgNzguMTc1NzgxIDE1Ni4xMDU0NjkgNzguOTA2MjUgQyAxNTcuNTc4MTI1IDgzLjIyNjU2MiAxNTcuNDA2MjUgODguMjE4NzUgMTU3LjQxNzk2OSA5Mi43NDYwOTQgQyAxNTcuNDIxODc1IDkyLjg4NjcxOSAxNTcuNDIxODc1IDkzLjAyNzM0NCAxNTcuNDIxODc1IDkzLjE2Nzk2OSBDIDE1Ny40MjU3ODEgOTMuNTY2NDA2IDE1Ny40MjE4NzUgOTMuOTY0ODQ0IDE1Ny40MjE4NzUgOTQuMzYzMjgxIEMgMTU3LjQyMTg3NSA5NC41MzkwNjIgMTU3LjQyMTg3NSA5NC41MzkwNjIgMTU3LjQyNTc4MSA5NC43MTg3NSBDIDE1Ny40MTc5NjkgOTUuMzAwNzgxIDE1Ny4zNjcxODggOTUuODE2NDA2IDE1Ny4yMTQ4NDQgOTYuMzc4OTA2IEMgMTU3LjA3ODEyNSA5Ni45MTQwNjIgMTU3LjAzOTA2MiA5Ny4yNzM0MzggMTU3LjA3ODEyNSA5Ny44MTI1IEMgMTU3LjA5NzY1NiA5OC45MjE4NzUgMTU2Ljc4MTI1IDk5Ljg3NSAxNTYuNDI5Njg4IDEwMC45MTc5NjkgQyAxNTYuMzc1IDEwMS4xMDE1NjIgMTU2LjMxNjQwNiAxMDEuMjg1MTU2IDE1Ni4yNTc4MTIgMTAxLjQ3NjU2MiBDIDE1NS44NzUgMTAyLjY0MDYyNSAxNTUuMzk0NTMxIDEwMy40NDUzMTIgMTU0LjUzMTI1IDEwNC4zMDg1OTQgQyAxNTMuNTQ2ODc1IDEwNS4zMDA3ODEgMTUzLjA0Njg3NSAxMDYuMjYxNzE5IDE1My4wMTE3MTkgMTA3LjY2Nzk2OSBDIDE1My4wMzUxNTYgMTA4LjQ0NTMxMiAxNTMuMTcxODc1IDEwOS4xOTE0MDYgMTUzLjMxMjUgMTA5Ljk1NzAzMSBDIDE1My40MTAxNTYgMTEwLjYwMTU2MiAxNTMuNDA2MjUgMTExLjIxMDkzOCAxNTMuMzgyODEyIDExMS44NTkzNzUgQyAxNTMuMTE3MTg4IDExMS45MTQwNjIgMTUyLjg1NTQ2OSAxMTEuOTY0ODQ0IDE1Mi41ODIwMzEgMTEyLjAxOTUzMSBDIDE1Mi41NDY4NzUgMTEyLjczNDM3NSAxNTIuNjMyODEyIDExMy4xOTUzMTIgMTUyLjkzMzU5NCAxMTMuODQzNzUgQyAxN
```

Comment:
I'm sorry, but it's still too big. This amount of data will significantly increase the load and cause delays on the web app's side.

Reply:
Ah, I see. The icon in the JSON file hasn't been updated yet.